README.md

```diff
@@ -1,17 +1,6 @@
 <p align="center">
   <h1 align="center">Talos</h1>
   <p align="center">A modern operating system for Kubernetes.</p>
-  <p align="center">
-    <a href="https://ci.dev.talos-systems.io/talos-systems/talos">
-      <img alt="Drone (self-hosted)" src="https://img.shields.io/drone/build/talos-systems/talos.svg?server=https%3A%2F%2Fci.dev.talos-systems.io&style=flat-square">
-    </a>
-    <a href="https://codecov.io/gh/talos-systems/talos">
```

Comment:
is that intentional to remove these badges?

Reply:
Yeah. I feel like this has little value to our users. Most of this stuck we are already made aware of by our CI tooling.